packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.test.tsx

```diff
@@ -91,7 +93,9 @@ describe('PaymentMethodTitle', () => {
             />,
         );
 
-        expect(component.find('[data-test="payment-method-name"]').text()).toEqual(
+        const component = screen.getByRole('heading');
+
+        expect(component?.textContent).toEqual(
             defaultProps.method.config.displayName,
         );
```

Comment:
Can I suggest using `toBeInTheDocument` instead of `toEqual` whenever possible for asserting text content in RTL.
```
expect(screen.getByRole('heading', { name: defaultProps.method.config.displayName })).toBeInTheDocument();
```